assets/training/finetune_acft_hf_nlp/environments/acpt-grpo/context/Dockerfile

```diff
@@ -0,0 +1,22 @@
+#PTCA image
+FROM mcr.microsoft.com/aifx/acpt/stable-ubuntu2204-cu126-py310-torch271:{{latest-image-tag:biweekly\.\d{6}\.\d{1}.*}}
+
+USER root
+
+RUN apt-get update && apt-get -y upgrade
+RUN pip install --upgrade pip
+
+COPY requirements.txt .
+RUN pip install -r requirements.txt --no-cache-dir
+
+RUN pip install azureml-acft-common-components=={{latest-pypi-version}}
+RUN pip install azureml-sdk==1.60.0
+RUN pip install numpy==2.2.5
+RUN pip install azureml-evaluate-mlflow=={{latest-pypi-version}}
+RUN pip install mlflow==3.1.0
```

Comment:
backend doesn't support mlflow>2, please make sure this work for your case

Reply:
We don't directly use mlflow, some package is pulling in as it's dependency and it needs to be updated because of vulnerability fix

Reply:
that would mean you overwrite that dependency with incompatible version

Reply:
we don't own the packages which needs to loosen or update dependency constraints. We've tested the section of code we need from them. There could be mis-compatibility or there could be nothing there, just a dependency constraint at setup.py will not decide the functional mis-compatibility